examples/with-next-js/pages/web-worker/index.tsx

```diff
@@ -0,0 +1,55 @@
+import Head from 'next/head'
+import React from 'react'
+import { Partytown } from '@builder.io/partytown/react'
+import { useWriteKey } from '../../utils/hooks/useConfig'
+
+const WebWorker: React.FC = () => {
+  const [writeKey] = useWriteKey()
+  return (
+    <>
+      <Head>
+        <Partytown
+          debug={true}
+          forward={[
+            'dataLayer.push',
+            'analytics.track',
+            'analytics.identify',
+            'analytics.group',
+            'analytics.reset',
+            'analytics.alias',
+          ]}
+        />
+        {/* copied the snippet from app.segment.com */}
+        <script
+          type="text/partytown"
```

Comment:
I looked at the partytown docs and _think_ I understand what's going on here...

So because of the script type, the JS is actually ran in a web worker right? Though party town will interact with the main thread's DOM and APIs as needed?

And then the `Partytown` component is just forwarding main thread `window.analytics` calls to the web worker?

If that's correct (or wrong!) might be helpful to add a comment or 2 on how this works.